src/components/ViewerTitle/index.tsx

```diff
@@ -58,9 +56,12 @@ const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
         );
 
     return (
-        <span className={styles.container}>
-            {title} {tag}
-        </span>
+        <div className={styles.container}>
+            <Popover content={title} mouseEnterDelay={1} mouseLeaveDelay={0}>
```

Comment:
So this popover shows the title even if the title is fully showing? Seems like we only want this if the title is truncated

Reply:
Yeah, it shows regardless. I agree it would be ideal to only show when the title is truncated, but I thought about the additional work to make that happen, and the possibility of putting additional information in the popover in the future, and instead just put in a 1 second delay for the tooltip appearing so it's not too annoying.  After I saw your comment I tried to make something work with useRef and useLayoutEffect and haven't had success yet. I'll give it another try tomorrow if you think this is really important!